lib/utils/timestamp.ts

```diff
@@ -0,0 +1,21 @@
+import { format } from 'date-fns';
+
+export class Timestamp {
+  constructor(private readonly date = new Date()) {}
+
+  public toString(dateFormat: string): string {
+    return format(this.#date(), dateFormat);
+  }
+
+  #date(): Date {
+    return new Date(
+      this.date.getUTCFullYear(),
+      this.date.getUTCMonth(),
+      this.date.getUTCDate(),
+      this.date.getUTCHours(),
+      this.date.getUTCMinutes(),
+      this.date.getUTCSeconds(),
+      this.date.getUTCMilliseconds()
+    );
+  }
+}
```

Comment:
would it be possible not to use a class here and just use a function?

Reply:
no problem, edited: https://github.com/mycodeself/mongo-migrate-ts/pull/99/commits/0dc0f02bc4e8c105950d3598a5f081e2313b68d5